### PR TITLE
Build tools versions updated

### PR DESCRIPTION
**Pull request by davyskiba**
Build tools versions updated to current version to fix build issues

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.6.21'
```
Comment (gawi151) [inline build.gradle]: Maybe we can bump it higher?

Comment (davyskiba) [reply build.gradle]: Updated to 1.9.24, which forced `kotlin-android-extension` removal

Comment (gawi151) [reply build.gradle]: Ok thx for doing the change. Yeah `kotlin-android-extension` was deprecated a long time ago :D

**Review by gawi151 — APPROVED** [review]
LGTM

**Pull request merged**